maestro-app/src/main/resources/config/application.yml

```diff
@@ -2,6 +2,9 @@ server:
   port: 11235
 
 maestro:
+  webclientConfig:
```

Comment:
it was better before, you could just refactor the java side, the word config here is unnecessary